### PR TITLE
Don't dynamically allocate memory for no good reason

### PR DESCRIPTION
**Pull request by htrefil**
Use a static const slice rather than needlessly allocating memory in a Vec.

### CODE REVIEW

**Comment by jheyens** [comment]
Hi, 
Thanks for the PR!

However, please do not commit format-only changes to the code in a feature PR.
This project has not yet [decided on an autoformat policy](https://github.com/greshake/i3status-rust/issues/340). Feel free to give your opinion on that issue!
I suggest you revert the changes that only target formatting. Feel free to create a format-only PR after the that issue resolves, though!

This will also make it easier for contributors to review your changes.

**Comment by htrefil** [comment]
Sorry, my bad, I did not want to include the formatting commit in this PR. It's fixed now.

File: src/blocks/cpu.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
-use crossbeam_channel::Sender;
 use crate::scheduler::Task;
 use crate::util::FormatTemplate;
+use crossbeam_channel::Sender;
```
Comment (jheyens) [inline src/blocks/cpu.rs]: This is still a formattting change

File: src/blocks/cpu.rs [hunk 2]
```diff
@@ -80,9 +80,8 @@ impl CpuConfig {
     }
 
     fn default_frequency() -> bool {
-      false
-  }
-
+        false
+    }
```
Comment (jheyens) [inline src/blocks/cpu.rs]: This is still a formattting change

File: src/blocks/cpu.rs [hunk 3]
```diff
@@ -103,8 +102,7 @@ impl ConfigBlock for Cpu {
             minimum_info: block_config.info,
             minimum_warning: block_config.warning,
             minimum_critical: block_config.critical,
-            format: FormatTemplate::from_string(&format)
-                .block_error("cpu", "Invalid format specified for cpu")?,
+            format: FormatTemplate::from_string(&format).block_error("cpu", "Invalid format specified for cpu")?,
```
Comment (jheyens) [inline src/blocks/cpu.rs]: This is still a formattting change

File: src/blocks/cpu.rs [hunk 4]
```diff
@@ -141,9 +139,14 @@ impl Block for Cpu {
...
-
             if line.starts_with("cpu") {
-                let data: Vec<u64> = (&line).split(' ').collect::<Vec<&str>>().iter().skip(if cpu_i == 0 { 2 } else { 1 }).filter_map(|x| x.parse::<u64>().ok()).collect::<Vec<_>>();
+                let data: Vec<u64> = (&line)
+                    .split(' ')
+                    .collect::<Vec<&str>>()
+                    .iter()
+                    .skip(if cpu_i == 0 { 2 } else { 1 })
+                    .filter_map(|x| x.parse::<u64>().ok())
+                    .collect::<Vec<_>>();
```
Comment (jheyens) [inline src/blocks/cpu.rs]: This is still a formattting change

File: src/blocks/cpu.rs [hunk 5]
```diff
@@ -171,7 +174,9 @@ impl Block for Cpu {
                 self.prev_idles[cpu_i] = idle;
                 self.prev_non_idles[cpu_i] = non_idle;
                 cpu_i += 1;
-                if cpu_i >= max_cpus { break; };
+                if cpu_i >= max_cpus {
+                    break;
+                };
```
Comment (jheyens) [inline src/blocks/cpu.rs]: This is still a formattting change

**Pull request closed without merging**